TestPullRequest/src/testPackage/CalcInterface2.java

```diff
@@ -2,7 +2,7 @@
 
 public interface CalcInterface2 extends CalcInterface3 {
 
-	public void testEclipse();
+	public void somethingCool();
```

Comment:
@LHommeDeBot Rename this method to voidMethod! For more feedback contact me at email@fakemail.de